Formula/libcouchbase.rb

```diff
@@ -29,7 +29,7 @@ def install
     args << "-DLCB_NO_TESTS=1"
 
     ["libev", "libevent", "libuv"].each do |pname|
-      args << "-DLCB_BUILD_#{pname.upcase}=" + (build.with?("#{pname}") ? "ON" : "OFF")
+      args << "-DLCB_BUILD_#{pname.upcase}=" + (build.with?(pname.to_s) ? "ON" : "OFF")
```

Comment:
`package_name` or even `pkgname` would occupy more line real estate that `pname`, though. How about `dep`?